docs/getting-started/installation-and-upgrade/installation-references/feature-flags.md

```diff
@@ -2,42 +2,38 @@
-- `token-hashing`: Used to enable new token-hashing feature. Once enabled, existing tokens will be hashed and all new tokens will be hashed automatically using the SHA256 algorithm. Once a token is hashed it cannot be undone. Once this feature flag is enabled, it cannot be disabled. See [hashing of tokens](../../../reference-guides/about-the-api/api-tokens.md) for more information.
-- `unsupported-storage-drivers`: This feature [allows unsupported storage drivers.](../../../how-to-guides/advanced-user-guides/enable-experimental-features/unsupported-storage-drivers.md). In other words, it enables types for storage providers and provisioners that are not enabled by default.
-- `istio-virtual-service-ui`: This feature enables a [UI to create, read, update, and delete Istio virtual services and destination rules,](../../../how-to-guides/advanced-user-guides/enable-experimental-features/istio-traffic-management-features.md) which are traffic management features of Istio.
-- `multi-cluster-management`: Used for multi-cluster provisioning and management of Kubernetes clusters. This feature flag can only be set at install time and not changed afterwards.
-
-The below table shows the availability and default value for feature flags in Rancher:
-
-| Feature Flag Name             | Default Value | Status       | Available as of | Rancher Restart Required? |
-| ----------------------------- | ------------- | ------------ | --------------- |---|
-| `istio-virtual-service-ui`    | `false`       | Experimental | v2.3.0          | |
-| `istio-virtual-service-ui`    | `true`        | GA*           | v2.3.2          | |
-| `unsupported-storage-drivers` | `false`       | Experimental | v2.3.0          | |
-| `fleet`  | `true` | GA* | v2.5.0 |   |
-| `fleet`  | `true` | Can no longer be disabled | v2.6.0 | N/A  |
-| `continuous-delivery` | `true` | GA* | v2.6.0 | |
-| `token-hashing` | `false` for new installs, `true` for upgrades | GA* | v2.6.0 | |
-| `legacy` | `false` for new installs, `true` for upgrades | GA* | v2.6.0 | |
-| `multi-cluster-management` | `false` | GA* | v2.5.0 | |
-| `harvester` | `true` | Experimental | v2.6.1 | |
-| `rke2` | `true` | Experimental | v2.6.0 | |
-
-\* Generally Available. This feature is included in Rancher and it is not experimental.
\ No newline at end of file
+The following is a list of feature flags available in Rancher v2.7. If you've upgraded from a previous Rancher version, you may see additional flags in the Rancher UI, such as `proxy` or `dashboard` (both [discontinued](../../../../versioned_docs/version-2.5/reference-guides/installation-references/feature-flags.md)):
```

Comment:
I think we should keep the first sentence generic and drop the mention of the version. For minor versions, we can omit explicit callouts since that can be inferred from the version being viewed. It also becomes another piece we need to update (and can be easily missed) when cutting a new version.